src/elements/table/Table.js

```diff
@@ -86,10 +88,11 @@ export class NovoTableHeader {
                                 </div>
                             </div>
                             <!-- FILTER DROP-DOWN -->
-                            <novo-dropdown side="right" *ngIf="column.filtering" class="column-filters">
+                            <novo-dropdown side="right" *ngIf="column.filtering" class="column-filters" [showAfterSelect]="column.range || column.multiple">
```

Comment:
The reason for this is clicking on the custom range it stays open to reveal the calendar, other options behave the same way as any other dropdowns